src/main/java/org/appdynamics/saaseng/Main.java

```diff
@@ -65,27 +65,39 @@ public static void main(String[] args) {
             logger.log(Level.SEVERE, "Client Exception : {}"+ e.getMessage());
         }
     }
-
+    
+    public static Subject buildSubject(String namespace) {
+        return new SubjectBuilder()
+                    .withApiGroup("rbac.authorization.k8s.io")
+                    .withKind("ServiceAccount")
+                    .withName("default")
+                    .withNamespace(namespace)
+                    .build();
+    }
+    
+    public static RoleDef buildRoleDef(){
```

Comment:
RoleDef to RoleRef defined in import io.fabric8.kubernetes.api.model.rbac.RoleRef;